configs/cityscapes/README.md

```diff
@@ -0,0 +1,33 @@
+# Benchmark and Model Zoo
+
+## Environment
+
+### Hardware
```

Comment:
This may be removed since we tested different hardwares and the performances are comparable.

Reply:
Agreed